static/library/donut_component.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { css, CSSResult, LitElement, unsafeCSS } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import _ from "lodash";
+import React from "react";
+import ReactDOM from "react-dom";
+
+import tilesCssString from "!!raw-loader!sass-loader!../css/tiles.scss";
+
+import {
+  DonutTile,
+  DonutTilePropType,
+} from "../js/components/tiles/donut_tile";
+import { DEFAULT_API_ENDPOINT } from "./constants";
+import { convertArrayAttribute } from "./utils";
+
+/**
+ * Web component for rendering a donut chart tile.
+ *
+ * Example usage:
+ *
+ * <!-- Show a donut chart of median income by gender in California -->
+ * <datacommons-donut
+ *      title="Median Income by gender in California"
+ *      place="geoId/06"
+ *      comparisonVariables='["Median_Income_Person_15OrMoreYears_Male_WithIncome", "Median_Income_Person_15OrMoreYears_Female_WithIncome"]'
+ * ></datacommons-donut>
+ */
+@customElement("datacommons-donut")
```

Comment:
Maybe call this `datacommons-pie` ? then in the future we can extend to support traditional pie charts as well as donut charts

Reply:
Renamed to `datacommons-pie`, and went ahead and enabled traditional pie charts since it's a small change. Drawing a pie chart is now default, but users can set the "donut" property to draw donuts instead. Updated the examples and screenshot in this PR's description -- PTAL.

Reply:
Nice!!